content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,133 @@
+
+Next is memory usage by pod. In both ambient scenarios, the L4 ztunnel consumes less memory than a sidecar. In fact, in the L4 only scenario, almost all three ztunnel instances combined consume less memory than a single sidecar. Ztunnels are able to serve multiple applications while consuming fewer resources than sidecars because they are purpose built, and designed to have a small footprint. For a more in depth explanation of how this is done, refer to this [blog](/blog/2023/rust-based-ztunnel/#a-purpose-built-ztunnel). Waypoint proxies consume a similar amount of resources as sidecars do, but once again see a minor increase due to the same reasons outlined above for CPU. So what do these per pod usages mean for the totals across the cluster? Everything.
+
+{{< image width="100%"
+    link="total-cpu.png"
+    caption="Total CPU usage for sidecar and ambient scenarios"
+    >}}
+
+{{< image width="100%"
+    link="total-memory.png"
+    caption="Total memory usage for sidecar and ambient scenarios"
+    >}}
+
+{{< image width="100%"
+    link="cpu-by-pod-stacked.png"
+    caption="Stacked CPU usage by pod for sidecar and ambient pods"
+    >}}
+
+{{< image width="100%"
+    link="memory-by-pod-stacked.png"
+    caption="Stacked memory usage by pod for sidecar and ambient pods"
+    >}}
+
+Looking at total CPU and memory utilization, we have to remember that in sidecar scenarios there are 144 sidecar containers required (48 services, 3 replicas each), while in ambient, only three ztunnel containers and 12 waypoint proxies are required. The stacked CPU and memory by pod graphs are excellent at highlighting just how many additional containers are present between scenarios. Memory usage of the Istio dataplane in the ztunnel-only ambient scenario sees *more than 99% savings* compared to sidecar scenarios, and 89% when waypoints are added. Looking at CPU, ztunnel sees *82% savings* compared to sidecar scenarios, and 40% when waypoints are deployed.
+
+Finally, let’s consider allocation since the graphs above have only covered usage. Every sidecar resource has a default request of 100 millicores vCPU and 128 MB memory. For simplicity, we assume ztunnels and waypoint proxies have similar requests and limits as their sidecar counterparts. Breaking it down, that’s a 98% reduction in allocated resources with ztunnel, and 90% when waypoints are included.
```

Comment:
> For simplicity, we assume ztunnels and waypoint proxies have similar requests and limits as their sidecar counterparts.

I think this sentence is doing a lot of heavy lifting.  Is this really likely?  Doesn't that talk to huge inefficiencies in the old method if so? Or, at least, inefficiencies that only apply at small enough scale?

(I don't presuppose the answer to this question)